static_src/components/users_parent_entity_user_selector.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Action from './action.jsx';
+import FormStore from '../stores/form_store';
+import { Form, FormSelect } from './form';
+import PanelDocumentation from './panel_documentation.jsx';
+import userActions from '../actions/user_actions';
+import { validateString } from '../util/validators';
+
+const USERS_PARENT_ENTITY_USER_FORM_GUID = 'users-parent-entity-users-form';
+
+const propTypes = {
+  userParentEntityUserSelectDisabled: PropTypes.bool,
+  currentUserAccess: PropTypes.bool,
+  parentEntityUsers: PropTypes.array,
+  error: PropTypes.object,
+  parentEntityGuid: PropTypes.string,
+  parentEntity: PropTypes.string,
+  currentEntityGuid: PropTypes.string,
+  currentEntity: PropTypes.string
+};
+const defaultProps = {
+  userParentEntityUserSelectDisabled: false,
+  currentUserAccess: false,
+  error: {}
+};
+
+function stateSetter(props) {
+  return {
+    userParentEntityUserSelectDisabled: props.userParentEntityUserSelectDisabled,
+    currentUserAccess: props.currentUserAccess,
+    parentEntityUsers: props.parentEntityUsers,
+    parentEntityGuid: props.parentEntityGuid,
+    parentEntity: props.parentEntity,
+    currentEntityGuid: props.currentEntityGuid,
+    currentEntity: props.currentEntity,
+    error: props.error
+  };
+}
+
+export default class UsersParentEntityUserSelector extends React.Component {
```

Comment:
I totally agree. Originally it was supposed to be a module. Not it only does one thing, check org users, if user is in a space.